YaLibs/YaToolsIDALib/Repository.cpp

```diff
@@ -275,10 +277,11 @@ namespace
         bool push(const std::string& src_branch, const std::string& dst_branch);
         bool remote_exist(const std::string& remote);
         std::string get_commit(const std::string& ref);
+        bool is_file_tracked(const std::string& path);
 
         GitRepo repo_;
         std::vector<std::string> comments_;
-        bool repo_auto_sync_;
+        bool repo_auto_sync_, include_idb_;
```

Comment:
split into one line per member

Reply:
fixed in 1d1e98c73c4b5c6eec04e07bac56ccb00b47b515